tools/reviewer/webapp/src/app/dashboard/file-changes/services/commit.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@angular/core';
+
+import { BranchInfo, Thread } from '@/shared/proto';
+import { LocalserverService, NotificationService } from '@/shared/services';
+import { StateService } from './state.service';
+import { ThreadService } from './thread.service';
+
+// Bunch of methods related to commits
+@Injectable()
+export class CommitService {
+  constructor(
+    private localserverService: LocalserverService,
+    private notificationService: NotificationService,
+    private stateService: StateService,
+    private threadService: ThreadService,
+  ) { }
+
+  getCurrentCommitIds(): void {
+    // Commit id before the changed. Left default value.
```

Comment:
we can check grammar here.

Reply:
I also don't understand this comment. Probably s/changed/change?
Also, what does this mean?
> Left default value.

Reply:
If left commit isn't set by UI/url then we use the default value.